medmodels/medrecord/tests/test_querying.py

```diff
@@ -609,6 +611,12 @@ def test_select_edges_edge(self):
             sorted(medrecord.select_edges(edge().connected("1"))),
         )
 
+        # Edge in group
```

Comment:
Do we get an invalid in_group test?

Reply:
any query that is syntactically correct (which is checked by pyright) is a valid query => there can be no test for an invalid query because they do not exist